docs/user-guide/configuration/annotations.md

```diff
@@ -36,4 +36,5 @@ Some Annotations are used to modify the properties of the build environment. The
 | beta.screwdriver.cd/cpu | `MICRO` / `LOW` / `HIGH` | When using a `k8s` executor, this will allow the user to choose between 0.5 (`MICRO`), 2 (`LOW`) and 6 (`HIGH`) CPU resources. Default is `LOW`.<br>   When using a `k8s-vm` executor, this will allow the user to choose between 1 (`MICRO`), 2 (`LOW`) and 6 (`HIGH`) CPU resources. Default is `LOW`. |
 | beta.screwdriver.cd/ram | `MICRO` / `LOW` / `HIGH` | When using a `k8s` executor, this will allow the user to choose between 1 GB (`MICRO`), 2 GB (`LOW`) and 12 GB (`HIGH`) RAM. Default is `LOW`.<br> When using a `k8s-vm` executor, this will allow the user to choose between 1 GB (`MICRO`), 2 GB (`LOW`) and 12 GB (`HIGH`) RAM. Default is `LOW`. |
 | beta.screwdriver.cd/timeout | Number of minutes | This will allow the user to choose the number of minutes a build should timeout after. Default is `90` minutes. |
-| screwdriver.cd/buildPeriodically | a CRON expression with an 'H' in the minutes field, e.g. `H 0 * * *` <br><br>**Note:** The first interval is always set to "H" (maximum frequency as once per hour) to avoid large spikes for shared resources. You can test out your cron expression at https://crontab.guru/ | This will trigger your job periodically according to the cron expression. |
+| screwdriver.cd/buildPeriodically | A CRON expression with an 'H' in the minutes field, e.g. `H 0 * * *` <br><br>**Note:** The first interval is always set to "H" (maximum frequency as once per hour) to avoid large spikes for shared resources. You can test out your cron expression at https://crontab.guru/ | This will trigger your job periodically according to the cron expression. |
+| screwdriver.cd/repoManifest | Git repository checkout URL, e.g. `https://github.com/org/repo.git/manifestFilePath.xml#branch` | [Repo](https://gerrit.googlesource.com/git-repo) is a tool built on top of Git for managing repositories. This value is the checkout URL of a Git repository that contains a repo manifest `xml` file (defaults to `default.xml`). If this value is specified, Screwdriver will checkout your source code in accordance with the configuration of the `xml` file. |
```

Comment:
Maybe it would be more clear if we specified that the value should be made up of three parts:
1. The git checkout url
2. The path to the manifest file
3. The git branch
Not sure what would be best

Reply:
I'll expand the definition to specify the parts :)